RequestService/RequestService.Core/Services/GroupService.cs

```diff
@@ -239,5 +243,29 @@ public async Task<GetGroupLocationsResponse> GetGroupLocations(int groupId)
                 return null;
             }
         }
+
+        public async Task<double?> GetGroupSupportActivityRadius(int groupID, SupportActivities supportActivity)
+        {
+            return await _memDistCache.GetCachedDataAsync(async (cancellationToken) =>
+            {
+                GetGroupSupportActivityRadiusRequest request = new GetGroupSupportActivityRadiusRequest() { GroupId = groupID, SupportActivityType = new SupportActivityType() { SupportActivity = supportActivity } };
+                string json = JsonConvert.SerializeObject(request);
+                StringContent data = new StringContent(json, Encoding.UTF8, "application/json");
+
+                using (HttpResponseMessage response = await _httpClientWrapper.PostAsync(HttpClientConfigName.GroupService, "/api/GetGroupSupportActivityRadius", data, CancellationToken.None))
```

Comment:
Really we should be passing through CancellationTokens, rather than using CancellationToken.None

Reply:
Added